packages/core/test-utils/src/utils.js

```diff
@@ -101,16 +101,21 @@ export async function bundle(
 }
 
 export async function run(
-  bundleGraph: BundleGraph,
+  bundleOrGraph: BundleGraph | Bundle,
   globals: mixed,
   opts: {require?: boolean} = {}
 ): Promise<mixed> {
-  let bundles = [];
-  bundleGraph.traverseBundles(bundle => {
-    bundles.push(bundle);
-  });
+  let bundle: Bundle;
+  if (bundleOrGraph.traverseBundles) {
```

Comment:
Flow doesn't like this, it should really be `bundleOrGraph instanceof BundleGraph` but that is obviously not possible.

This change was needed because the `bundleGraph`'s entry bundle can be a html file but I want to execute the included JS script.